test/unit/discovery_attribute_set_test.rb

```diff
@@ -1,4 +1,5 @@
-require 'test_helper'
```

Comment:
I actually can't that does not work, this requires `test_helper` from Foreman and not discovery. It's a mess.